recipe/meta.yaml

```diff
@@ -76,7 +77,7 @@ requirements:
     - pango
```

Comment:
libiconv run_exports is not applied. Not sure why, but we should at least set pin_compatible in the run section. 
see https://staging.continuum.io/package/prefect/fs/librsvg-feedstock/pr9/75f8f18/osx-64/librsvg-2.56.3-hf076d44_0.tar.bz2

Reply:
Yeah, that's weird. I added `{{ pin_compatible('libiconv') }}` with a comment.
I have a feeling that all `libiconv` downstreams have the same problem.

Fixed.
Before: `"libiconv",` https://staging.continuum.io/package/prefect/fs/librsvg-feedstock/pr9/0a2fe7a/osx-arm64/librsvg-2.56.3-h44e7d58_0.tar.bz2
After:   `"libiconv >=1.16,<2.0a0"`, https://staging.continuum.io/package/prefect/fs/librsvg-feedstock/pr9/4fbd1ea/osx-arm64/librsvg-2.56.3-h44e7d58_0.tar.bz2